### DISCUSSION TITLE
Is there anybody who can help on a prerender error?

### DISCUSSION THREAD

**Discussion by Kiteguide** [vercel/next.js#40102] [Help]
Dear Community

After having our developers investigating for weeks and trying to get along with AWS support, we are still stuck in Code Build with a prerender error. I am not a programmer, but I understand that either we are blind, dumb or simply not experienced enough with next.js to get across the error. Is there anybody who can help live resolving this issue? I am willing to pay by the hour. Maybe it takes minutes, who knows 🤷‍♂️😀. Would be very appreciated. 

`<html>
<body>

Error occurred prerendering page "/". Read more: https://nextjs.org/docs/messages/prerender-error
--
152 | Error: Bad Gateway
153 | at /codebuild/output/src549227869/src/github.com/kiteguide-gmbh/kiteguide-2/packages/webapp/.next/server/chunks/79.js:84:23
154 | at processTicksAndRejections (internal/process/task_queues.js:95:5)
155 | at async Promise.all (index 6)
156 | at async getStaticProps (/codebuild/output/src549227869/src/github.com/kiteguide-gmbh/kiteguide-2/packages/webapp/.next/server/pages/index.js:527:28)
157 | at async renderToHTML (/codebuild/output/src549227869/src/github.com/kiteguide-gmbh/kiteguide-2/packages/webapp/node_modules/next/dist/server/render.js:388:20)
158 | at async /codebuild/output/src549227869/src/github.com/kiteguide-gmbh/kiteguide-2/packages/webapp/node_modules/next/dist/export/worker.js:273:36
159 | at async Span.traceAsyncFn (/codebuild/output/src549227869/src/github.com/kiteguide-gmbh/kiteguide-2/packages/webapp/node_modules/next/dist/trace/trace.js:74:20)
160 | info  - Generating static pages (38/152)
161 | info  - Generating static pages (76/152)
162 | info  - Generating static pages (114/152)
163 | info  - Generating static pages (152/152)

</body>
</html>`

**Comment by gaston-flores** [comment]
Hey @Kiteguide. Going by the message it'd seem that a fetch in `getStaticProps` is throwing that error. Are you fetching a publicly available resource or something inside a VPC? Can you share some example code or a reproducible example? Do you only see this issue when building through CodeBuild?

**Comment by Kiteguide** [comment]
Something inside a VPC
Via an Api (node.js lambda), the api throws a bad gateway error, logs indicate that the api cannot reach the database (timeout), but aws aurora direct access via aws panel works (-> database itself works). It stopped around the time we added an ssl certificate and served the website via our main domain. Does this make sense?
